src/libmongoc/src/mongoc/mongoc-cluster.c

```diff
@@ -1067,41 +1067,20 @@ _mongoc_cluster_get_auth_cmd_x509 (const mongoc_uri_t *uri,
    return false;
 #else
    const char *username_from_uri = NULL;
-   char *username_from_subject = NULL;
 
    BSON_ASSERT (uri);
 
    username_from_uri = mongoc_uri_get_username (uri);
    if (username_from_uri) {
       TRACE ("%s", "X509: got username from URI");
-   } else {
-      if (!ssl_opts || !ssl_opts->pem_file) {
```

Comment:
The `ssl_opts` parameter is now unused and can be removed from `_mongoc_cluster_get_auth_cmd_x509`.

The `error` parameter also now needs a `BSON_UNUSED`.